app/views/service_providers/_form.html.erb

```diff
@@ -78,9 +78,9 @@
   <%= form.input :default_aal,
                  as: :select,
                  collection: [
-                   ['', nil],
-                   ['AAL2', 2],
-                   ['AAL2 + Phishing-Resistant MFA', 3]],
+                   ['MFA required + remember device up to 30 days (AAL1)', 1],
```

Comment:
**issue:** I don't think we should change the value to `1` here - with the way the IdP works, `nil` represents our default behavior. Funnily enough we do technically support a value of `1` but we never use it (since it wasn't available in the dropdown) - AAL1 is technically weaker than our default since it doesn't include MFA.